src/components/timer.js

```diff
@@ -0,0 +1,69 @@
+/*  this is a timer-component */
+import React, { Component } from 'react'
+import '../App.css'
+import SimpleTable from './table'
+import { Grid } from '@material-ui/core'
+import MenuBar from './menu'
+/* inserting table data */
+const tableHeader = { 1: 'Name', 2: 'Age', 3: 'Gender', 4: 'Member', 5: 'Email', 6: 'Status' }
```

Comment:
column name member needs to be changed or it should be a meaning full  column name